### DISCUSSION TITLE
目录中的点点线太长，与页码重叠了，请问怎么缩短一点

### DISCUSSION THREAD

**Discussion by xbzeng1010** [sjtug/SJTUThesis#917] [提问]
![screenshot-20230411-161045](https://user-images.githubusercontent.com/33973762/231097890-9313c5f5-7b9b-4b6c-9c83-bb7b1a6e206c.png)

**Comment by LogCreative** [comment]
一般而言，前言部分不必在目录中体现，所以采用了宽度默认值。但是罗马数字宽度较大，所以在导言区添加下面的配置 [^1]：
```latex
\cftsetpnumwidth{2em} % 1.55em
```
[^1]: https://tex.stackexchange.com/a/186561/253485
